src/bf_dpu_update.py

```diff
@@ -93,6 +94,8 @@ def _http_post(self, url, data, headers=None, timeout=(60, 60)):
     def _http_patch(self, url, data, headers=None, timeout=(60, 60)):
         return self.http_accessor(url, 'PATCH', self.username, self.password, headers, timeout).access(data)
 
+    def _http_put(self, url, data, headers=None, timeout=(60, 60)):
```

Comment:
We have two http accessor: curl and requrests.
Current, we use curl by default, it's why _http_put can be supported auto.
But, if we switch to use requests, _http_put will failed.

Please help to also provide PUT method in http_accessor_requests.py
    def access(self, data=None):
        if self.method == 'GET':
            return self._http_get()
        elif self.method == 'POST':
            return self._http_post(data=data)
        elif self.method == 'PATCH':
            return self._http_patch(data=data)

Reply:
Done